src/redux/categories/categories.js

```diff
@@ -1,22 +1,20 @@
-// ACTIONS
-const MESSAGE = 'bookstore/categories/MESSAGE';
+import { createSlice } from '@reduxjs/toolkit';
 
 // INITIAL STATE
-const initialState = [];
+const initialState = {
+  status: [],
+};
 
-// ACTION CREATORS
-export function updateAction() {
-  return {
-    type: MESSAGE,
-  };
-}
+const categorySlice = createSlice({
+  name: 'category',
+  initialState,
+  reducers: {
+    displayMessage: (state) => {
+      state.status = 'Under constructions'; //eslint-disable-line
```

Comment:
- [ ] Kindly avoid disabling linters as it is not a good practice, it is better for your linters to pass the linter checks 👍🏿

Reply:
@prolajumokeoni I tried but the syntax of the redux toolkit is showing linter errors

Reply:
- [ ]  Kindly share the error message you get, you can also google the error message to see how others have fixed it 👍🏿